mirrord/cli/src/main.rs

```diff
@@ -129,6 +129,10 @@ where
             }
         }
     }
+
+    // Kills the intproxy, freeing the agent.
+    execution_info.stop().await;
```

Comment:
1. clippy.
2. please move above the `if`, so that we don't `return` without killing the intproxy.

Reply:
Changed Mr. Wackz.